cell.go

```diff
@@ -114,21 +114,21 @@ func (c *Cell) SetFloatWithFormat(n float64, format string) {
 
 var timeLocationUTC, _ = time.LoadLocation("UTC")
 
-func timeToUTCTime(t time.Time) time.Time {
+func TimeToUTCTime(t time.Time) time.Time {
```

Comment:
My bad, changes from d5dc146f4ea75e6b34d1a79324cedb22dd708157 on should have been in another branch/pull request altogether, I didn't expect this pull request to show them. 
It's the only way I found to set custom date formats using SetDateTimeWithFormat as a user that would be consistent with what's already there. Granted those 2 if exported could use a move to a more "timely" package. ^^
I also saw another pull request for that which looked more thought through, those changes were meant to stay on my repo for this one time I needed them.
Feel free to ignore those particular changes.